EnvironmentDefinitions/ARMTemplates/App_Service_with_Cosmos/azuredeploy.json

```diff
@@ -0,0 +1,3743 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.21.1.54444",
+      "templateHash": "10809287839753264304"
+    }
+  },
+  "parameters": {
+    "repoUrl": {
+      "type": "string",
+      "allowedValues": [
+          "https://github.com/azure-samples/todo-java-mongo",
+          "https://github.com/azure-samples/todo-nodejs-mongo",
```

Comment:
The API app service does not appear to be coming online for me. `azd up` command completed successfully.

Reply:
I selected the node template, but after deployment discovered the runtime stack was set to Java.